src/lib/alerts.ts

```
@@ -0,0 +1,16 @@
+interface AlertResponse {
+  id: number
+  quote: string
+  author: string
+}
+
+export const getAlert = async (): Promise<string | undefined> => {
+  try {
+    const response = await fetch('https://dummyjson.com/quotes/random')
+    const data: AlertResponse = await response.json()
+    return 'Arkade Bulletin Board System goes here'
+    return data.quote
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)